frontend/src/pages/org/settings/components/visibility.ts

```diff
@@ -34,19 +34,19 @@ export class OrgSettingsVisibility extends BtrixElement {
               ?checked=${this.org?.enablePublicProfile}
               ?disabled=${!this.org}
             >
-              ${msg("Allow anyone to view org")}
+              ${msg("Enable gallery of public collections")}
             </sl-switch>
           </div>
         `,
         msg(
-          "If enabled, anyone on the Internet will be able to view general org information and browse this org's public collections.",
+          "If enabled, anyone on the Internet will be able to browse this org's public collections and view general org information.",
         ),
       ],
       [
         html`
           <div class="mb-2">
             <btrix-copy-field
-              label=${msg("Public Page URL")}
+              label=${msg("Gallery URL")}
```

Comment:
Might keep the language "public gallery" here?  I like re-enforcing that it's viewable by the public!
```suggestion
              label=${msg("Public Gallery URL")}
```